### DISCUSSION TITLE
Grafana and next 12(ish) hours of data values

### DISCUSSION THREAD

**Discussion by thenecroscope** [BottlecapDave/HomeAssistant-OctopusEnergy#185] [Q&A] [answered]
Hi,

I'm new(ish) to both HomeAssistant and this Addon. Addon is working great.
Real quick question, I've enabled the Grafana Addon within HomeAssistant. I can see historical data points, current and next rate. 
Is it possible to read in the next 12 hours of values. Is this stored in another field somewhere?

Many thanks,
Steve

**Comment by fboundy** [comment, marked as answer]
They are an attribute so you would need to make a template sensor for each that you want to be recorded into the historical database. Look up Home Assistant Templating (stateattr is the function you need in Jinja2).

You may be aware that InfluxDB is probably a better long term database for using with Grafana than the default SQLlite. Plenty of tutorials out there as to how to set it up.

**Comment by thenecroscope** [comment]
Thanks for replying. I will have a play around.

Thanks again

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
